Dockerfiles/agent/amd64/Dockerfile

```diff
@@ -29,20 +29,19 @@ FROM baseimage AS extract
 ARG WITH_JMX
 ARG PYTHON_VERSION
 ARG DD_AGENT_ARTIFACT=datadog-agent*_amd64.deb
+ARG GENERAL_ARTIFACTS_CACHE_BUCKET_URL
 
 # copy everything - globbing with args wont work
 COPY datadog-agent*_amd64.deb /
 WORKDIR /output
 
-# Get s6-overlay and check gpg signature
-# Hack: PGP key store locally as it's currently not available from https://keybase.io/justcontainers/key.asc
-COPY hack/s6-pgp-key /tmp/s6-pgp-key
+# Get s6-overlay
```

Comment:
could add an explaination about how to update the S6_SHA256SUM ?

Reply:
Done ✅